README.adoc

```diff
@@ -194,11 +190,11 @@ verbatimModuleSyntax:: Controls how TypeScript preserves your module import and
 +
 Setting this to *true* ensures your imports work exactly as written. _(This is particularly important when working with Foundry's module system, as unexpected transformations of import statements can cause hard-to-debug runtime errors.)_
 
-forceConsistentCasingInFileNames:: Ensures that when you import files, the casing in your import statement must exactly match the actual filename. For example, `import "./MyFile"` won't work if the file is actually named `myfile.ts`.
+forceConsistentCasingInFileNames:: Ensures that casing in imported files must exactly match the actual filename. For example, `import "./MyFile"` won't work if the file is actually named `myfile.ts`.
 +
-Setting this to *true* prevents bugs that could occur when your project runs on case-sensitive systems. _(This is especially important if you're developing on Windows but your users might be running Foundry on Linux, where `MyFile.ts` and `myfile.ts` are treated as different files.)_
+Setting this to *true* prevents bugs that could occur when your project runs on case-sensitive systems. _(This is especially important if anyone contributing to your project develops on Windows where imports _are_ case insensitive because your users might be running Foundry on Linux, where `MyFile.ts` and `myfile.ts` are treated as different files.)_
 
-resolveJsonModule:: Allows you to import JSON files directly as modules.
+resolveJsonModule:: Allows importing JSON files directly for automatic types.
```

Comment:
I decided I wanted to avoid saying `module` here since it seemed to vague to someone who doesn't already know the term.

Automatic types is less vague without being overly verbose.